justfile

```diff
@@ -23,4 +25,10 @@ build:
     colcon build
 
 run-agent:
-    ros2 run fetch_agent agent
\ No newline at end of file
+    ros2 run fetch_agent agent
+
+all:
```

Comment:
dont do that. These are not things you always want to run

you can add a build & run if you want but you dont want to isntall deps everytime and fix lints...